docs/04-operation-guides/troubleshooting/apix-03-404-not-found.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cannot connect to a service exposed by an API Rule - 404 Not Found
+---
+
+## Symptom
+
+You reach your service and get `404 Not Found` in response.
+
+## Remedy
+
+In such case, make sure that:
+
+- Proper Oathkeeper Rule has been created:
+
+  ```bash
+  kubectl get rules.oathkeeper.ory.sh -n {NAMESPACE}
+  ```
+
+  >**TIP:** Name of the Rule consists of the name of the API Rule and a random suffix.
+
+- Proper VirtualService has been created:
+
+  ```bash
+  kubectl get virtualservices.networking.istio.io -n {NAMESPACE}
+  ```
+
+  >**TIP:** Name of the VirtualService consists of the name of the API Rule and a random suffix.
+
+Sometimes Oathkeeper Maester controller stops reconciling Rules on long-living clusters. This can result in random `404 Not Found` responses, because Oathkeeper does not contain Rules reflecting the actual state of the cluster. A simple restart of the Pod resolves the issue, but you might want to verify if that is the issue you have encountered. To do so:
+
+1. Fetch all Oathkeeper Pods' names:
+
+    ```bash
+    kubectl get pods -n kyma-system -l app.kubernetes.io/name=oathkeeper -o jsonpath='{.items[*].metadata.name}'
+    ```
+
+2. Fetch the access Rules from every Oathkeeper Pod and save them to a file:
+
+    ```bash
+   kubectl cp -n kyma-system -c oathkeeper "{POD_NAME}":etc/rules/access-rules.json "access-rules.{POD_NAME}.json" 
+   ```
+
+3. If you have more than one instance of Oathkeeper, compare whether the files contain the same Rules. Oathkeeper stores Rules as JSON files, so you might want to use [jd](https://github.com/josephburnett/jd) to automate the comparison:
+
+    ```bash
+   jd -set {FIRST_FILE} {SECOND_FILE} 
+   ```
+
+    Oathkeeper Pods are out of sync if there are any differences between the files other than the order of Rules.
+   
+4. Compare the Rules in the files with Rules present on the cluster. If there are any differences, Oathkeeper Pods are out of sync.
```

Comment:
These sentences are kind of a duplication. I get that in the first one you want to explain what actually counts as a "difference". How about rewording the sentences as such? 
```suggestion
    The files are considered different by jd if there are any differences between the files other than the order of Rules.
   
4. Compare the Rules in the files with Rules present on the cluster. If the files are different, Oathkeeper Pods are out of sync.
```

Or maybe you have a better idea?